CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+## 1.6.3
```

Comment:
we typically set the version as part of releasing, depending on what changes. since this is a new feature, we will want to bump the minor.

```suggestion
```